docs/src/reducers/componentState/StickyState.ts

```diff
@@ -0,0 +1,41 @@
+import { IDocument } from '../../Types';
+import StickyExample from '../../examples/StickyExample/StickyExample';
+import StickyExample1 from '../../examples/StickyExample/StickyExample1';
+
+const StickyExampleCode = require('!raw-loader!../../examples/StickyExample/StickyExample') as string;
+const StickyExample1Code = require('!raw-loader!../../examples/StickyExample/StickyExample1') as string;
+
+const StickyState: IDocument = {
+  id: 'Sticky',
+  heading: 'Sticky',
+  subheading: `Make elements remain at the top or bottom of the viewport, like a sticky navigation.`,
+  property: [
+    {
+      name: 'children',
+      type: 'any',
+      desc: 'Elements to display inside Sticky.',
+    }, {
+      name: 'position',
+      type: 'Position',
+      desc: 'Position prop allow user to choose StickyHeader or StickyFooter',
+    }, {
+      name: 'style',
```

Comment:
@HiirenP We are using `componentStyle` key for style props

Reply:
resolved